frontend/src/components/crop/Navbar.tsx

```diff
@@ -80,6 +96,35 @@ export default function ImageNavbar({
             />
           </Tooltip>
         </Grid2>
+        <Grid2>
+          <Typography>Selection Mode</Typography>
+          <Box display="flex" justifyContent="center" sx={{ padding: "5px" }}>
+            <Select
+              labelId="demo-simple-select-label"
+              id="demo-simple-select"
+              value={
+                selectionMode === SelectionMode.Single ? "Single" : "Multi"
+              }
+              label="Age"
+              onChange={(e) => {
+                if (e.target.value === "Single") {
+                  setSelectionMode(SelectionMode.Single);
+                } else if (e.target.value === "Multi") {
+                  setSelectionMode(SelectionMode.Multi);
+                }
+              }}
+            >
+              <MenuItem value={"Single"}>Single</MenuItem>
+              <MenuItem value={"Multi"}>Multi</MenuItem>
+            </Select>
+            <Tooltip
+              title="Single selection mode only allows one rectangular region of interest for all sample images.
+            Multi selection mode allows separate regions to be used in different images"
+            >
```

Comment:
This covers almost all the info that I think a user may want to know, but I reckon mentioning something about how the multi-selection mode takes a union of the ROIs selected for the different images will be useful to mention - otherwise I think it's not clear how the user is intended to use the multi-selection to get a _single_ ROI selection out of it.